modules/client-configuration/pages/channels.adoc

```diff
@@ -54,6 +54,8 @@ For more information about content lifecycle management, see xref:administration
 For {redhat} based clients, additional packages are available through AppStream.
 In most cases, the AppStream packages are required to ensure that you have all the software you need.
 
+When you are managing AppStream packages in the {productname} {webui}, you might notice that you see contradicting suggestions for package updates.
+This is due to the {productname} not always interpreting the modular metadata correctly.
```

Comment:
Hmm, "not always interpreting the modular metadata correctly" doesn't sound right here. SUMA doesn't interpret the modular metadata at all, and the suggestions are expected to be wrong.

The basic idea is:
If you have modular packages in the list, do not expect SUMA UI to display correct stuff because it doesn't interpret modular metadata.

Reply:
edited